### PR TITLE
feat(storage): Turbo Replication Samples added

### PR DESCRIPTION
**Pull request by saranshdhingra**
Issue #1536

### CODE REVIEW

File: storage/src/create_bucket_turbo_replication.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ *
+ * @param string $bucketName The name of your Cloud Storage bucket.
+ * @param string $location The Dual Region location where you want your bucket to reside. (Read more at https://cloud.google.com/storage/docs/locations#location-dr)
+ */
+function create_bucket_turbo_replication($bucketName, $location = 'nam4')
+{
+    // $bucketName = 'my-bucket';
+
+    $storage = new StorageClient();
+    $rpo = 'ASYNC_TURBO';
```
Comment (bshaffer) [inline storage/src/create_bucket_turbo_replication.php:41]: Is this not a Proto constant somewhere? It seems like it should be...

File: storage/src/set_turbo_replication_async_turbo.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ */
+function set_turbo_replication_async_turbo($bucketName)
+{
+    // $bucketName = 'my-bucket';
+
+    $storage = new StorageClient();
+    $bucket = $storage->bucket($bucketName);
+
+    $bucket->update([
+        'rpo' => 'ASYNC_TURBO'
```
Comment (bshaffer) [inline storage/src/set_turbo_replication_async_turbo.php:43]: maybe for consistency, set this to a `$rpo` variable like in the first example?

File: storage/src/set_turbo_replication_default.php [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+{
+    // $bucketName = 'my-bucket';
+
+    $storage = new StorageClient();
+    $bucket = $storage->bucket($bucketName);
+
+    // Updating the rpo value of a multi-region bucket to DEFAULT has no effect
+    // and updating the rpo value of a regional bucket will throw an exception.
+    $bucket->update([
+        'rpo' => 'DEFAULT'
```
Comment (bshaffer) [inline storage/src/set_turbo_replication_default.php:44]: Same here...

**Review by bshaffer — APPROVED** [review]
this looks great! Just a few suggestions

**Comment by bshaffer** [comment]
Looks like you have an error in your assertion: 
```
--

There was 1 failure:

1) Google\Cloud\Samples\Storage\Tests\TurboReplicationTest::testCreateBucketWithTurboReplication
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'ASYNC_TURBO'
+'DEFAULT'

/tmpfs/src/github/php-docs-samples/storage/test/TurboReplicationTest.php:62

--

```

**Comment by saranshdhingra** [comment]
> Looks like you have an error in your assertion:
> 
> ```
> --
> 
> There was 1 failure:
> 
> 1) Google\Cloud\Samples\Storage\Tests\TurboReplicationTest::testCreateBucketWithTurboReplication
> Failed asserting that two strings are equal.
> --- Expected
> +++ Actual
> @@ @@
> -'ASYNC_TURBO'
> +'DEFAULT'
> 
> /tmpfs/src/github/php-docs-samples/storage/test/TurboReplicationTest.php:62
> 
> --
> ```

Hey @bshaffer 
Thanks for the review :) 
Yes, I expect this error to stay until the library isn't released.
The [PR](https://github.com/googleapis/google-cloud-php/pull/4543) for the library is ready but still has the `Do not merge` label.

**Review by ddelgrosso1 — CHANGES_REQUESTED** [review]
The region tags for samples should be:

storage_create_bucket_turbo_replication
storage_get_rpo
storage_set_rpo_async_turbo
storage_set_rpo_default

File: storage/src/create_bucket_turbo_replication.php (hunk 1 above)
Comment (saranshdhingra) [reply storage/src/create_bucket_turbo_replication.php:41]: @dwsupplee and I discussed that in the library PR previously.
https://github.com/googleapis/google-cloud-php/pull/4543

I previously had used class constants, but then we changed it to regular strings.
Nothing in the proto though.

**Pull request merged**